modules/virt-importing-vm.adoc

```diff
@@ -0,0 +1,132 @@
+. Select *Connect to New Instance* or a saved RHV instance from the *RHV Instance* list.
+
+* If you select *Connect to New Instance*, fill in the following fields:
+
+** *API URL*: For example, `\https://<RHV_Manager_FQDN>:8443/ovirt-engine/api`
+** *CA certificate*: Click *Browse* to upload the RHV Manager CA certificate.
+** *Username*: RHV Manager admin username
+** *Password*: RHV Manager admin password
+
+* If you select a saved RHV instance, the wizard connects to the RHV instance using the saved credentials.
+endif::[]
+
+. Click *Check and Save* and wait for the connection to complete.
+
+ifdef::virt-importing-rhv-vm[]
+. Select a cluster and a virtual machine to import.
+endif::[]
+ifdef::virt-importing-vmware-vm[]
+. Select a virtual machine or a template to import.
+endif::[]
+
+. Click *Next*.
+
+. In the *Review* screen, review your settings.
+ifdef::virt-importing-rhv-vm[]
++
+Optional: You can select *Start virtual machine on creation*. The RHV virtual machine is powered off before it is imported.
+endif::[]
+
+. If the settings are correct, click *Import*.
++
+A *Successfully created virtual machine* message and a list of resources created for the virtual machine are displayed. The virtual machine appears in *Workloads* -> *Virtual Machines*.
+
+. Optional: Click *Edit* to update the following settings:
+
+* *General*:
+
+** *Name*: If the imported virtual machine name is already being used by another virtual machine in the namespace, update the name.
+** *Description*: Optional
+** *Operating System*
+** *Flavor*
+** *Memory*
+** *CPUs*
+** *Workload Profile*
+
+* *Networking*:
+
+** *Name*
+** *Model*
+** *Network*
+** *Type*
+** *MAC Address*
++
+Optional: Click *Add Network Interface* to add an interface.
+
+* *Storage*:
+
+** *Name*
+** *Source*: For example, *Import Disk*
+** *Size*
+** *Interface*
+ifdef::virt-importing-rhv-vm[]
+** *Storage Class*: If you do not select a storage class, {VirtProductName} uses the default `nfs-storage` storage class to create the virtual machine.
+endif::[]
+ifdef::virt-importing-vmware-vm[]
+** *Storage Class*: If you do not select a storage class, {VirtProductName} uses the default storage class to create the virtual machine.
+endif::[]
+** *Advanced* > *Volume Mode*
+** *Advanced* > *Access Mode*
++
+Optional: Click *Add Disk* to add a disk to the virtual machine.
+
+ifdef::virt-importing-vmware-vm[]
+* *Advanced* > *Cloud-init*:
+
+** *Form*: Enter the *Hostname* and *Authenticated SSH Keys*.
+** *Custom script*: Enter the `cloud-init` script in the text field.
+endif::[]
+ifdef::virt-importing-rhv-vm[]
+* *Advanced* > *Cloud-init*: `Not Enabled`
++
+`cloud-init` is not imported with the RHV source virtual machine. You must recreate the `cloud-init` settings after import.
+endif::[]
+* *Advanced* > *Virtual Hardware*: You can attach a virtual CD-ROM to the imported virtual machine.
+
+. If the settings are correct, click *Review and Import*.
++
```

Comment:
> The powered off virtual machine appears in *Workloads* -> *Virtual Machines*.

for vmware yes.

for rhv 
- firstly `VirtualMachineImport` resource is displayed (as I mentioned in https://github.com/openshift/openshift-docs/pull/22423#discussion_r435329439) with pending status
- then once some validations pass `VirtualMachine` is displayed or error status shown
- then progress is displayed (error status can still occur from here)
- once import completes the vm may or may not start depending on the wizard configuration

not sure how much we want to document this behaviour

Reply:
I don't think we need to document this entire process.

If a RHV VM imports successfully, will it appear in Workloads > Virtual Machines or not?

Reply:
yup, it will